### PR TITLE
Improve the Italian translation

### PR DESCRIPTION
**Pull request by Palladinium**
My little bit of help with #23. I'm a native Italian speaker but not an Italian Pokemon fan, so I don't know if there are better accepted terms in the community.

### CODE REVIEW

File: translating_strings/Italiano.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ English: Pseudo Legend
 Translation: Pseudo Leggendario
 
 English: Box Legendary
-Translation: Scatola leggendaria
```
Comment (Palladinium) [inline translating_strings/Italiano.txt:8]: "Scatola leggendaria" reads as "legendary box". No idea what the proper term for this is, but at least "Leggendario di scatola" reads as "legendary of the box" which is still an improvement.

**Comment by Palladinium** [comment]
There's also the issue with "Favorite" being translated as "Preferito". Unfortunately, Italian has grammatical gender, so "preferito" is only correct with masculine nouns (like fossile) but not feminine nouns (like Ultrabestia), which needs "preferita". Fixing that would require reworking the translation string, so I'll leave it out of this PR since it'll affect a lot more than Italian.

**Comment by Palladinium** [comment]
"Modulo espediente" also doesn't make a whole lot of sense to me, but I have no better alternative.

**Comment by CajunAvenger** [comment]
> There's also the issue with "Favorite" being translated as "Preferito". Unfortunately, Italian has grammatical gender, so "preferito" is only correct with masculine nouns (like fossile) but not feminine nouns (like Ultrabestia), which needs "preferita". Fixing that would require reworking the translation string, so I'll leave it out of this PR since it'll affect a lot more than Italian.

Add Preferita to the feminine nouns and i'll get the gender scripts set up for Italian. (or put Preferito on the masculine if that's easier)

**Comment by SteveNerd22** [comment]
> "Modulo espediente" also doesn't make a whole lot of sense to me, but I have no better alternative.In the Italian translation, 


'Gimmick form' is translated as 'Modulo espdiente.' However, the current name is incorrect, as 'Modulo' means 'Module,' not 'Form' as it should be intended. I suggest translating it as 'Forma speciale' (Special form) instead.

**Pull request merged**